sort/quick_sort.py

```diff
@@ -1,17 +1,25 @@
-def quick_sort(arr, first, last):
-    """ Quicksort
+def quick_sort(arr, first=0, last=-999):
```

Comment:
This looks fine. But the version with the single argument `arr` was better. I meant to say that you make changes in the test cases in tests/test_sort.py, and remove those extra arguments from the quicksort call.